packages/ui/react-ui-theme/src/styles/layers/tag.css

```diff
@@ -0,0 +1,73 @@
+@layer dx-components {
+  .dx-tag {
+    @apply text-xs font-semibold pli-2 plb-0.5 rounded cursor-default truncate;
+    user-select: none;
+
+    &[data-hue='neutral'] {
+      @apply bg-separator text-neutral-700 dark:text-neutral-150;
```

Comment:
Maybe this combination of neutrals should become a sememe… leave a todo for now perhaps?